src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalSum = days * 40;
+
+  if ((days >= 3) && (days < 7)) {
+    totalSum -= 20;
+  };
+
+  if (days >= 7) {
+    totalSum -= 50;
+  };
```

Comment:
what is `20` and `50`? add some meaningful constants

Reply:
I fix this